pkgs/games/glulxe/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "glulxe";
+  version = "0.5.4";
+
+  buildInputs = [ ncurses ]; 
+
+  srcs = [
+    (fetchurl {
+      url = "https://eblong.com/zarf/glulx/glulxe-054.tar.gz";
+      sha256 = "0vipydg6ra90yf9b3ipgppwxyb2xdhcxwvirgjy0v20wlf56zhhz";
+    })
+    (fetchurl {
+      url = "https://eblong.com/zarf/glk/glkterm-104.tar.gz";
+      sha256 = "0zlj9nlnkdlvgbiliczinirqygiq8ikg5hzh5vgcmnpg9pvnwga7";
+    })
+  ];
+
+  sourceRoot = "glulxe";
+
+  configurePhase = ''
+  substituteInPlace Makefile \
+    --replace "#GLKINCLUDEDIR = ../glkterm" "GLKINCLUDEDIR = ../glkterm" \
+    --replace "#GLKLIBDIR = ../glkterm" "GLKLIBDIR = ../glkterm" \
+    --replace "#GLKMAKEFILE = Make.glkterm" "GLKMAKEFILE = Make.glkterm"
+
+    cd ../glkterm
+    make
```

Comment:
Why is make invoked in the configure phase? I suppose you want to invoke it in the build phase? In that case, use `preBuild` and `postBuild` to change folder, and keep the default `buildPhase`, to ensure `make` gets invoked with `makeFlags` etc.

Reply:
Annoyingly to me as well, the Makefile in glkterm is acting more like "configure" than a build: it is producing the Make.glkterm which is then used by the build of glulxe.